plugins/source/aws/client/spec/account.go

```diff
@@ -24,22 +24,22 @@ type Account struct {
 	//   ```
 	//
 	// `local_profile` should be set to either `default` or `user1`.
-	LocalProfile string `json:"local_profile,omitempty"`
+	LocalProfile string `json:"local_profile,omitempty" jsonschema:"example=my_aws_profile"`
 
 	// If specified will use this to assume role.
 	RoleARN string `json:"role_arn,omitempty" jsonschema:"pattern=^(arn(:[^:\n]*){5}([:/].*)?)?$"`
 
 	// If specified will use this session name when assume role to `role_arn`.
-	RoleSessionName string `json:"role_session_name,omitempty"`
+	RoleSessionName string `json:"role_session_name,omitempty" jsonschema:"example=my_aws_role_session_name"`
 
 	// If specified will use this when assuming role to `role_arn`.
-	ExternalID string `json:"external_id,omitempty"`
+	ExternalID string `json:"external_id,omitempty" jsonschema:"example=external_id"`
 
 	// If specified, this region will be used as the default region for the account.
-	DefaultRegion string `json:"default_region,omitempty" jsonschema:"minLength=1,default=us-east-1"`
+	DefaultRegion string `json:"default_region,omitempty" jsonschema:"minLength=1,default=us-east-1,example=us-east-1"`
```

Comment:
We don't need to add examples for fields that have a default, as the default serves as an example